OpenUpSA/wazimap-ng-ui
# displaying point summary after visiting a geo with zero points

## Description
The bug this PR fixes : 
* navigate to #geo:582005001
* in the rich data panel the point summary is hidden because there are zero points
* navigate to #ZA
* even though there are points now, the point summary is still hidden


## Related Issue
https://wazimap.atlassian.net/browse/WNCM-147

## How is it tested automatically?
added more steps into `__tests__/gui/facilities.feature`

## How should a reviewer test it locally
* navigate to #geo:582005001
* in the rich data panel confirm that the point summary is hidden because there are zero points
* navigate to #ZA
* confirm that the point summary is not still hidden


## Screenshots
![image](https://user-images.githubusercontent.com/53019884/144067706-e827ba67-8656-4ef9-a191-1c1f4c39a5a8.png)


## Changelog

### Added

### Updated
* `src/js/profile/facilities/facility_controller.js` to remove the `hidden` class when there are points in the current geography

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
